test/dash/dash_parser_manifest_unit.js

```diff
@@ -1898,11 +1868,13 @@ describe('DashParser Manifest', () => {
     fakeNetEngine.setResponseText('dummy://foo', manifestText);
     const config = shaka.util.PlayerConfiguration.createDefault().manifest;
     config.dash.manifestPreprocessor = (mpd) => {
-      const selector = 'AdaptationSet[mimeType="text/vtt"';
-      const vttElements = mpd.querySelectorAll(selector);
-      for (const element of vttElements) {
-        element.parentNode.removeChild(element);
-      }
+      /** @type {shaka.extern.xml.Node} */
+      const manifest = /** @type {shaka.extern.xml.Node} */ (
+        /** @type {shaka.extern.xml.Node} */(mpd).children[0]);
```

Comment:
nit: add two spaces in the indentation

Reply:
The linter reverts to this spacing unfortunately.

Reply:
ok!